bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueBridgeHandler.java

```diff
@@ -238,7 +238,7 @@ protected void doConnectedRun() throws IOException, ApiException {
             Map<String, FullGroup> lastGroupStateCopy = new HashMap<>(lastGroupStates);
 
             for (final FullGroup fullGroup : hueBridge.getGroups()) {
-                State groupState = fullGroup.getState();
+                State groupState = new State();
```

Comment:
Why a new state?

Reply:
Because this state is not returned by the API.

In fact, for groups, the hue bridge is returning a state object but this is a different object than the one we map in class State (the state for lights). This is the reason why there was no NPE with the hue bridge. But with the hue emulation binding, fullGroup.getState() is null and this leads to a NPE few lines below.

Reply:
understood